src/ucs/arch/rv64/global_opts.h

```diff
@@ -0,0 +1,29 @@
+/**
+* Copyright (C) Tactical Computing Labs, LLC. 2022. ALL RIGHTS RESERVED.
+*
+* See file LICENSE for terms.
+*/
+
+#ifndef UCS_RV64_GLOBAL_OPTS_H_
+#define UCS_RV64_GLOBAL_OPTS_H_
+
+#include <stddef.h>
+
+#include <ucs/sys/compiler_def.h>
+
+BEGIN_C_DECLS
+
+#define UCS_ARCH_GLOBAL_OPTS_INITALIZER {   \
+    .builtin_memcpy_min = UCS_MEMUNITS_AUTO, \
+    .builtin_memcpy_max = UCS_MEMUNITS_AUTO  \
+}
+
+/* built-in memcpy config */
+typedef struct ucs_arch_global_opts {
+    size_t builtin_memcpy_min;
+    size_t builtin_memcpy_max;
+} ucs_arch_global_opts_t;
```

Comment:
can remove builtin memcpy options for rv64, it's not used anyway
for example aarch64 does not have it

Reply:
thought this was removed previously, looks like the last push didn't remove it.